packages/rancher-monitoring/rancher-monitoring/generated-changes/patch/templates/alertmanager/servicemonitor.yaml.patch

```diff
@@ -1,16 +1,10 @@
 --- charts-original/templates/alertmanager/servicemonitor.yaml
 +++ charts/templates/alertmanager/servicemonitor.yaml
-@@ -39,10 +39,20 @@
-     tlsConfig: {{ toYaml .Values.alertmanager.serviceMonitor.tlsConfig | nindent 6 }}
+@@ -39,7 +39,16 @@
+     tlsConfig: {{- toYaml .Values.alertmanager.serviceMonitor.tlsConfig | nindent 6 }}
      {{- end }}
      path: "{{ trimSuffix "/" .Values.alertmanager.alertmanagerSpec.routePrefix }}/metrics"
--{{- if .Values.alertmanager.serviceMonitor.metricRelabelings }}
-     metricRelabelings:
--{{ tpl (toYaml .Values.alertmanager.serviceMonitor.metricRelabelings | indent 6) . }}
--{{- end }}
-+    {{- if .Values.alertmanager.serviceMonitor.metricRelabelings }}
-+    {{ tpl (toYaml .Values.alertmanager.serviceMonitor.metricRelabelings | indent 6) . }}
-+    {{- end }}
+-    {{- if .Values.alertmanager.serviceMonitor.metricRelabelings }}
```

Comment:
Historically our relabelling indents have been a source of indent-based bugs, should double check that this is correctly indented

Reply:
Looks fine to me:
```
    metricRelabelings:
      - action: keep
        regex: kube_(daemonset|deployment|pod|namespace|node|statefulset).+
        sourceLabels:
        - __name__
```